File: Python/168.java
class Solution {
    public String convertToTitle(int columnNumber) {
        String s = "";
        while(columnNumber != 0){
            s = (char)((columnNumber - 1)%26 + 65) + s;
            if(columnNumber%26 == 0){
                columnNumber -= 26;
            }
            columnNumber /= 26;
        }
        return s;
    }
}

File: Python/168.py
class Solution(object):
    def convertToTitle(self, columnNumber):
        s = ""
        while(columnNumber):
            s = chr((columnNumber-1)%26 + 65) + s
            columnNumber -= 26 if s[0] == 'Z' else 0
            columnNumber /= 26
        return s